### ISSUE TITLE
ReferenceError: value is not defined

### ISSUE THREAD

**Issue by baxterio** [SSSaaS/sssa-js#2]
Hello,

I'm getting this error when using Brave v1.73.101 / Chromium: 131.0.6778.139

```javascript
Uncaught (in promise) ReferenceError: value is not defined
```

You need to declare the `value` variable correctly

https://github.com/SSSaaS/sssa-js/blob/9a2a5338398d81aa23ae62b3dc73f23eaa33b67e/sssa.js#L296
```javascript
let value = utils.random();
```

https://github.com/SSSaaS/sssa-js/blob/9a2a5338398d81aa23ae62b3dc73f23eaa33b67e/sssa.js#L309
```javascript
let value = utils.random();
```

Regards

**Comment by cipherboy** [comment]
Hello @baxterio, happy to take a PR. Do note that sssa-js was written over 9 years ago, when ES5 was still popular, and had no updates since. :-)

**Comment by baxterio** [comment]
yes, it's true that back then it wasn't a problem! Ok I'll open a pr soon then!
